st2api/st2api/config.py

```diff
@@ -28,10 +28,23 @@
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
+def parse_args(args=None):
+    CONF(args=args, version=VERSION_STRING)
+
+
+def register_opts():
+    _register_common_opts()
+    _register_app_opts()
+
+
 def _register_common_opts():
     common_config.register_opts()
 
 
+def get_logging_config_path(service='api'):
```

Comment:
I decided to do the refactoring and move `resultstracker` and `notifier` in a package under `st2actions/`.

Eventually, we will probably want to do a bigger refactoring and move those to packages to the top level outside of `st2actions/`.

On a related note - during this refactoring, I also noticed some bad coupling between scheduler / re-scheduler - currently both actionrunner and notifier depend on it and use it.

When we are doing a bigger refactoring we should also fix this coupling.